apps/Electron Fiddle/install-32

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+fiddle_ver="electron-fiddle_0.27.0_armhf.deb"
```

Comment:
```suggestion
```

Reply:
Why? This just makes updating Fiddle easier.

Reply:
The variable is only used one time in the script. We can just simply change the version number at 
`install_packages https://github.com/electron/fiddle/releases/download/v0.27.2/electron-fiddle_0.27.2_armhf.deb || exit 1`.

So I don't think defining a variable is needed.